networkx/drawing/nx_pydot.py

```diff
@@ -182,8 +168,10 @@ def to_pydot(N, strict=True):
     -----
 
     """
-    pydot = load_pydot()
-
+    try:
+        import pydotplus as pydot
+    except ImportError:
+        raise ImportError("pydotplus required for to_pydot()")
```

Comment:
This is a lot of code repeat for very little gain. I'd prefer to continue using `load_pydot` just to handle the try/except, or not do a try/except at all and let the standard `ImportError` raise.